### PR TITLE
Feature/neo11

### PR DESCRIPTION
**Pull request by fmina**
@ffurrer

### CODE REVIEW

File: rotors_gazebo/resource/neo11.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# NEO11 Hex vehicle parameters
+mass: 3.42
+inertia: {xx: 0.0608 , xy: 0.0, xz: 0.0, yy: 0.0688, yz: 0.0, zz: 0.1489}
+
+# angle should be expressed in rad
+# direction 1 means counter clockwise, -1 clockwise
+# The order of the rotors matters for the calculation of the allocation matrix.
+rotor_configuration:
+  '0': {angle: 0.52359877559, arm_length: 0.2895, rotor_force_constant: 1.269e-05, rotor_moment_constant: 1.6754e-2, direction: 1.0}
```
Comment (burrimi) [inline rotors_gazebo/resource/neo11.yaml]: are you sure about the moment constant? seems very high

File: rotors_description/urdf/neo11_base.xacro [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<robot name="neo11" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find rotors_description)/urdf/component_snippets.xacro" />
+  <!-- Instantiate euroc_hex "mechanics" -->
```
Comment (ffurrer) [inline rotors_description/urdf/neo11_base.xacro]: change this to neo too ;)

File: rotors_description/urdf/neo11_vi_sensor.gazebo [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<robot name="neo11" xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <!-- Instantiate default euroc_hex-->
```
Comment (ffurrer) [inline rotors_description/urdf/neo11_vi_sensor.gazebo]: neo

Comment (ffurrer) [inline rotors_description/urdf/neo11_vi_sensor.gazebo]: maybe also check the other files from the last PR.

File: rotors_description/urdf/neo11_generic_odometry_sensor.gazebo [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
```
Comment (burrimi) [inline rotors_description/urdf/neo11_generic_odometry_sensor.gazebo]: I think this file is no longer needed. there is a mav_generic_odometry_sensor.gazebo file for all mavs.

Comment (ffurrer) [inline rotors_description/urdf/neo11_generic_odometry_sensor.gazebo]: true, thanks :)

File: rotors_gazebo/resource/neo11.yaml (hunk 1 above)
Comment (fmina) [inline rotors_gazebo/resource/neo11.yaml]: The torque is given by rotor_moment_constant* rotor_force_constant*omega^2. This close enough to firefly (1.6e-2). Why do you think it is too high?

Comment (burrimi) [inline rotors_gazebo/resource/neo11.yaml]: sorry, my bad. Didn't remember that we multiply it with the force constant in rotors.

**Pull request merged**